src/redux/stock/actionCreator.js

```diff
@@ -0,0 +1,40 @@
+import { API_URL, API_KEY } from 'react-native-dotenv';
+
+import { FETCH_STOCK_DETAILS_START, FETCH_STOCK_DETAILS_FAIL, FETCH_STOCK_DETAILS_SUCCESS } from './actionType';
+
+const fetchStockDetailsStart = () => ({
+  type: FETCH_STOCK_DETAILS_START,
+});
+
+const fetchStockDetailsFail = (err) => ({
+  type: FETCH_STOCK_DETAILS_FAIL,
+  payload: err,
+});
+
+const fetchStockDetailsSuccess = (data) => ({
+  type: FETCH_STOCK_DETAILS_SUCCESS,
+  payload: data,
+});
+
+// eslint-disable-next-line import/prefer-default-export
+export const fetchStockDetails = (symbol) => (dispatch) => {
```

Comment:
If this file could be extended later then I would suggest to export an object at the end of the file, so later we dont need to remove this eslint configuration when we add more exports.
On the other hand, if you dont want to extend this file, then please use `export default`. 

```js
export default {
  fetchStockDetails,
}
```

Reply:
Updated